verbcalc/core/symbols.py

```diff
@@ -42,6 +44,10 @@ def divisions(self) -> list:
     def exponentiations(self) -> list:
         return self._exponentiations
 
+    @property
+    def absolutes(self) -> list:
+        return self._taking_absolute
```

Comment:
Always make sure to use 'refactor' option in your IDE instead of just changing name of something. In this instance, you have changed the name of self._taking_absolute to self._absolutes, but this change is only reflected in __init__, not here in line 49.
```suggestion
        return self._absolutes
```